src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -323,7 +323,14 @@ public void commitAddressBook() {
 
     @Override
     public double getCap() {
-        return versionedTranscript.getCurrentCap();
+        double cap = round(versionedTranscript.getCurrentCap(), 1);
+        return cap;
+        //return versionedTranscript.getCurrentCap();
+    }
+
+    private static double round (double value, int precision) {
```

Comment:
And this method is duplicated right? I think we can do some reusing here.

Reply:
@jeremiah-ang yep right you are, should be all fixed now!